src/sequence_processing_pipeline/Pipeline.py

```diff
@@ -911,8 +911,8 @@ def is_sample_sheet(sample_sheet_path):
 
         return False
 
-    def _generate_dummy_sample_sheet(self, first_read, last_read,
-                                     indexed_reads, dummy_sample_id):
+    def _generate_dummy_sample_sheet(self, index_cycles, non_index_cycles,
```

Comment:
FWIW, the code expected (as this is how old RunInfo.xml files are formed) that the reads would be like index1, read1, read2, index2 - however this is not the case in the new versions. Thus, changed the method to actually be aware of what's an index and non_index read

Reply:
I am 100% in favor of anything that makes the nature of the inputs more explicit! However, I don't really understand what the inputs should be here. Is it possible to add a docstring for this function, maybe with an example input for each param?